Makefile

```diff
@@ -54,6 +54,10 @@ shell-% sh-%: ## Run a shell in a container
 test: ## Run tests
 	npx jest ./src --verbose true
 
+.PHONY: integration-test
+integration-test: ## Run integration tests locally
+	npx jest ./tests/api_e2e --verbose true
+
```

Comment:
I would like to keep this. It's very convenient to run the integration tests locally from time to time without using Docker.